MTA10/mods/shared_logic/CClientPed.cpp

```diff
@@ -5576,12 +5576,37 @@ void CClientPed::SetDoingGangDriveby ( bool bDriveby )
         }
         else if ( bDriveby )
         {
-            bool bRight = ( GetOccupiedVehicleSeat ( ) % 2 == 0 ) ? false : true;
+            char cSeat = GetOccupiedVehicleSeat ( );
+            bool bRight = ( cSeat % 2 == 0 ) ? false : true;

```

Comment:
cSeat % 2 == 0 is already a boolean expression.

cSeat % 2 != 0 has the same effect without the ternary operator